docs/src/build-from-source.md

```diff
@@ -30,7 +30,7 @@ git submodule update --init
 ./scripts/setup/ubuntu/install_deps.sh
 ./scripts/setup/ubuntu/install_cbmc.sh
 ./scripts/setup/ubuntu/install_viewer.sh
-# If you haven't already:
+# If you haven't already (or from [rustup](https://rustup.rs/)):
```

Comment:
I don't know if you saw my comment on the original PR: https://github.com/model-checking/kani/pull/2093#discussion_r1065042035

Reply:
ack. fixed. thanks.